### PR TITLE
config for Renoir APU

### PR DESCRIPTION
**Pull request by perexg**
This is just a test config (not for merge).

BEWARE: This configuration is not intended for the broad use at the moment. It will broke other HDA Intel hardware.

### CODE REVIEW

File: ucm2/HDA-Intel/Hdmi.conf [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+				JackControl "HDMI/DP,pcm=3 Jack"
+			}
+		}
+	}
+}
+
+If.hdmi2 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=4 Jack'"
```
Comment (jason77-wang) [inline ucm2/HDA-Intel/Hdmi.conf:32]: The hdmi2 on that amd renoir machine is "iface=CARD,name='HDMI/DP,pcm=7 Jack'"

File: ucm2/HDA-Intel/Hdmi.conf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch',index=1 off"
+			]
+
+			Value {
+				PlaybackPriority 600
+				PlaybackPCM "hw:${CardId},4"
+				JackControl "HDMI/DP,pcm=4 Jack"
```
Comment (jason77-wang) [inline ucm2/HDA-Intel/Hdmi.conf:49]: Change the 4 to 7
				PlaybackPCM "hw:${CardId},7"
				JackControl "HDMI/DP,pcm=7 Jack"

File: ucm2/HDA-Intel/Hdmi.conf [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+			}
+		}
+	}
+}
+
+If.hdmi3 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=5 Jack'"
```
Comment (jason77-wang) [inline ucm2/HDA-Intel/Hdmi.conf:58]: The hdmi3 on that amd renoir machine is 8 instead 5: "iface=CARD,name='HDMI/DP,pcm=8 Jack'"

File: ucm2/HDA-Intel/Hdmi.conf [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch',index=2 off"
+			]
+
+			Value {
+				PlaybackPriority 700
+				PlaybackPCM "hw:${CardId},5"
+				JackControl "HDMI/DP,pcm=5 Jack"
```
Comment (jason77-wang) [inline ucm2/HDA-Intel/Hdmi.conf:75]: And change 5 to 8:

			        PlaybackPCM "hw:${CardId},8"
				JackControl "HDMI/DP,pcm=8 Jack"

**Comment by perexg** [comment]
I tried to resolve all issues in dcef48f13d4f5db79b006755074940b94730a883 (master). It needs testing on the real hardware, of course.

**Pull request closed without merging**